README.adoc

```diff
@@ -31,36 +29,23 @@ feedback, feature requests, or just want to meet for some drinks and talk
 software design patterns, contact us either at the developer portal or via
 email: mailto:api@smartrac-group.com[api@smartrac-group.com]
 
-
-== SMART COSMOS Objects License
-You must review and accept the
-https://licensing.smartcosmos.net/objects/[SMART COSMOS Objects EULA] before
-working with this developer kit.
-
-== Open Source Framework
-SMART COSMOS includes a number of open source frameworks available under the
-Apache License, Version 2. You can clone and freely use these frameworks in your
-projects, submit pull requests, and access the latest documentation at GitHub:
-
- * https://github.com/SMARTRACTECHNOLOGY/smartcosmos-framework
-
- * https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit
-(where this document was downloaded from)
```

Comment:
I would remove this, as it can also be read on the Github directly, or a generated PDF, or whatever.